incubator/sugarcrm/values.yaml

```diff
@@ -0,0 +1,98 @@
+## Bitnami sugarcrm image 
+## ref: https://hub.docker.com/r/bitnami/sugarcrm/tags/
+##
+image: bitnami/sugarcrm
+
+## Specify a imagePullPolicy
+## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+##
+# imagePullPolicy:
+
+## User of the application
+## ref: https://github.com/bitnami/bitnami-docker-sugarcrm#environment-variables
+##
+sugarcrmUsername: user
+
+## User of the application
```

Comment:
`User of the application` -> `Last name`. I wonder if the sugarCRM image is missing a first name param.